bookwyrm_backend/src/main/java/com/bookwyrm/backend/book/controller/BookController.java

```diff
@@ -21,12 +21,21 @@
 
```

Comment:
This whole change should be moved into a service file, because a large amount of logic is happening here. 

You'll have to make a service.impl file and extend the BookService. If that doesnt work, then you can make an interface/impl pair for SortService.